jenkins/add-docker-build-job.sh

```diff
@@ -40,6 +42,8 @@ job_short_name="basic-docker-build"
 job_display_name="Basic Docker Build"
 job_description="A basic pipeline that builds a Docker container. The job expects a Dockerfile at the root of the git repository"
 repository="${USER}/myfirstapp"
+scm_poll_schedule=""
```

Comment:
We'll add the in the quickstart template. In the script it will be an optional argument